cmd/kube-scheduler/app/server.go

```diff
@@ -417,13 +420,20 @@ func NewSchedulerServer(config *componentconfig.KubeSchedulerConfiguration, mast
 		metricsServer = makeMetricsServer(config)
 	}
 
+	// Prepare the number of workqueu parallel, validate NumberOfWorkQueueParallel
+	// in configuration file, or use default number of parallelism(16).
+	if config.NumberOfWorkQueueParallel <= 0 {
```

Comment:
Should not set the default value here.

The default value is applied here.

https://github.com/kubernetes/kubernetes/blob/b3ec8295be792cf752873f3781369204d3b05ea0/cmd/kube-scheduler/app/server.go#L156-L165

Reply:
Is default value set to `GOMAXPROCS` better?

Reply:
16 is the current use of the value, I do not know why it was the value, so keep this default.

Reply:
@wojtek-t could you help to answer the question about the default `16` workers? Thanks! :smiley: